server/recording_api.go

```diff
@@ -15,6 +15,57 @@ import (
 
 var callRecordingActionRE = regexp.MustCompile(`^\/calls\/([a-z0-9]+)/recording/(start|stop|publish)$`)
 
+const recordingJobStartTimeout = 15 * time.Second
+
```

Comment:
Actually, the timeout I was considering here would be the HTTP client one which was taken care of in https://github.com/mattermost/mattermost-plugin-calls/pull/249 through bumping dependencies since this checker goroutine only starts once the HTTP request returns so that means the recording job (docker container) has started and 15 seconds maps well to the [existing one](https://github.com/mattermost/calls-recorder/blob/dd30b8b78c6d704394dcfc46de584df01fe30afe/cmd/recorder/recorder.go#L23) in `calls-recorder`.